Freshli.Web/Controllers/AccountController.cs

```diff
@@ -0,0 +1,48 @@
+using System.Threading.Tasks;
+using Freshli.Web.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Freshli.Web.Controllers {
+  public class AccountController : Controller {
+
+    private UserManager<IdentityUser> userManager;
+
+    private SignInManager<IdentityUser> signInManager;
+
+    public AccountController(UserManager<IdentityUser> userMgr,
+      SignInManager<IdentityUser> signInMgr) {
+      userManager = userMgr;
+      signInManager = signInMgr;
```

Comment:
Instead of shortening `userManager` to `userMgr`, I think we should avoid ambiguity by either renaming the private fields to start with `_` characters, or we should use `this.` to clear things up.

```suggestion
    public AccountController(UserManager<IdentityUser> userManager,
      SignInManager<IdentityUser> signInManager) {
      _userManager = userManager;
      _signInManager = signInManager;
```

Alternatively,

```suggestion
    public AccountController(UserManager<IdentityUser> userManager,
      SignInManager<IdentityUser> signInManager) {
      this.userManager = userManager;
      this.signInManager = signInManager;
```